# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/kdissert/abs_is_stdlib_h.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN kdissert-1.0.5/src/kdissert/canvasview/DCanvasLink.cpp kdissert-1.0.5-new/src/kdissert/canvasview/DCanvasLink.cpp
--- kdissert-1.0.5/src/kdissert/canvasview/DCanvasLink.cpp	2005-10-22 02:43:14.000000000 +0400
+++ kdissert-1.0.5-new/src/kdissert/canvasview/DCanvasLink.cpp	2005-11-15 02:43:35.000000000 +0300
@@ -13,6 +13,7 @@
  ***************************************************************************/
 
 #include <math.h>
+#include <stdlib.h>
 #include <qpainter.h>
 #include <qcolor.h>
 #include <kdebug.h>
